pootle/apps/pootle_store/utils.py

```diff
@@ -166,37 +155,27 @@ def accept_suggestion(self, suggestion):
         # important to set these attributes after saving Submission
         # because in the `ScoreLog` we need to access the unit's certain
         # attributes before it was saved
-        unit.submitted_by = suggestion_user
+        # THIS NEEDS TO GO ^^
```

Comment:
I would use this PR to add the data migration that sets suggestion submitters to `nobody` for those suggestions without submitter.